### PR TITLE
Js use real dbref

### PR DESCRIPTION
**Pull request by jsingle**
@inlinestyle this is what I think we want... Version bump waiting on tom

### CODE REVIEW

**Comment by jsingle** [comment]
Tests are already breaking haha

File: tests/helper_tests.py [hunk 1]
```diff
@@ -135,7 +135,8 @@ def test_dbref_to(self, get_factory, create):
         get_factory.return_value = factory
         create.return_value = {"_id": 1234}
         func = dbref_to("user")
-        self.assertEqual({"$id": 1234, "$ref": "users"}, func())
+        dbref = func()
+        self.assertEqual({"$id": 1234, "$ref": "users"}, {'$id': dbref.id, '$ref': dbref.collection})
```
Comment (inlinestyle) [inline tests/helper_tests.py]: I'd change this assertion to be against an actual DBRef, rather than converting/comparing it to a dictionary

**Comment by inlinestyle** [comment]
@jsingle the main part of the code looks good, had a change I'd like you to make for the tests.

**Comment by inlinestyle** [comment]
@jsingle looks good to merge :space_invader:

**Pull request merged**